cmd/utrace/run/cmd.go

```diff
@@ -34,21 +34,27 @@ func init() {
 		"log-level",
 		"l",
 		`log level, options: panic, fatal, error, warn, info, debug or trace`)
+	Utrace.Flags().BoolVarP(
+		&options.GenerateGraph,
+		"generate-graph",
+		"g",
+		false,
+		`when set, utrace will generate a .dot graph with the collected statistics`)
 	Utrace.Flags().VarP(
```

Comment:
In our quest to manage PIE / Non-PIE execs, we had some trouble managing the binary option. As we use the proc maps to symbolise, not knowing where to start from in the proc maps was troublesome.

Perhaps the limitation around this could be that you need to specify the binary that matches the symbol. Then the pids that use this symbol would be traced. This also means that we would need to be able to build the symbol cache in a lazy way (instead of filling the cache from the start)

@nsavoire fyi